tests/testthat/test_extractData2.R

```diff
@@ -198,6 +198,42 @@ test_that("extractData2 with some variables labels applied to (convertVariables
                           b = c(2, 1), stringsAsFactors = TRUE))
 })
 
+test_that("Extract data trend GADS", {
+  # trend_gads <- getTrendGADS(filePaths = c("tests/testthat/helper_dataBase.db", "tests/testthat/helper_dataBase_uniqueVar.db"), years = c(2012, 2018), fast = FALSE)
+  trend_gads <- suppressWarnings(getTrendGADS(filePaths = c("helper_dataBase.db", "helper_dataBase_uniqueVar.db"),
+                                              years = c(2012, 2018), fast = FALSE, verbose = FALSE))
+  out <- extractData2(trend_gads)
+  expect_equal(dim(out), c(6, 5))
+  expect_equal(names(out), c("ID1", "V1", "V2", "V3", "year"))
+  comp <- c(rep(2012, 3), c(rep(2018, 3)))
+  attr(comp, "label") <- "Trendvariable, indicating the year of the assessment"
+  expect_equal(out$year, comp)
+
+  ## convertVariables if some variables are not in both GADS
+  out2 <- extractData(trend_gads, convertVariables = "V3")
+  expect_equal(out, out2)
```

Comment:
Nit (?): Yields the same result if `convertVariables = "V3"` is omitted (V3 seems to be treated equally across `extractData()` and `extractData2()`?

Reply:
Fixed (should have been extractData2). Modified the test, so `V3` has actually value labels that can be applied.